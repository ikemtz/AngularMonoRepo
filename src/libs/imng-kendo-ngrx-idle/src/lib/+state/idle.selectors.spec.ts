import { idleQuery } from './idle.selectors';

describe('Idle Selectors', () => {
  it('should select the feature state', () => {

    expect(idleQuery).toMatchSnapshot();
  });
});
