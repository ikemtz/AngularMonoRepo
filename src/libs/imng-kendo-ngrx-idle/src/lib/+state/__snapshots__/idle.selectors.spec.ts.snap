// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Idle Selectors should select the feature state 1`] = `
Object {
  "getIsTimingOut": [Function],
  "getTimeoutSpanInMs": [Function],
}
`;
